docs/mctl_list_firmware-set.md

```diff
@@ -5,16 +5,16 @@
 List firmware
 
 ```
-mctl list firmware-set [flags]
+mctl list firmware-set -v VENDOR -m MODEL [flags]
 ```
 
 ### Options
 
 ```
   -h, --help                    help for firmware-set
-  -l, --labels stringToString   Labels to from the firmware set - 'foo=bar,foo2=bar2' (default [])
-  -m, --model string            filter by model
-  -v, --vendor string           filter by vendor
+  -l, --labels stringToString   Labels to identify the firmware set - e.g. 'key=value,default=true,latest=true' (default [])
+  -m, --model string            [required] filter by model
+  -v, --vendor string           [required] filter by vendor
```

Comment:
It's not, the labels are optional. We were selecting for latest and default silently.

Reply:
Vendor and model have always been required, cf. https://github.com/metal-toolbox/fleetdb/blob/d3c9080fe47ca110f415ec0236c572c546a224cc/pkg/api/v1/router_firmware_set.go#L70

Reply:
🤔 Let me try something. Maybe I can make this work.

Reply:
There's a weird interaction with the FleetDB API for listing firmware. If you specify neither vendor nor model, you get a list of the first N firmware sets (currently 47, that's all that's in FleetDB). If you specify _either_ vendor or model you *must* specify both or the FleetDB API throws an error as shown in my link above. We can argue about what the correct FleetDB behavior should be, but ultimately that's out of scope for this change right now.

I think that in most cases, a user of `mctl` has a notion of the vendor and model for the firmware set they want to install, and because of that I think it would be less surprising to a user to require these flags in all cases. 

What do you think?